sag_desc.par

```diff
@@ -45,8 +45,8 @@ $
 7:11-kleisma (896/891)
 Poseidon's trident
 ~|(
-g
-17 (or 245 or 7:13) -comma (4131/4096)
+e
+17-comma (4131/4096)

```

Comment:
The 7:13 comma does not appear to be a thing anymore. While the 245 comma _is_ a secondary comma of this symbol, why call this one out especially? Why wouldn't every one of these symbols list all of its secondary commas? Aren't there potentially infinite secondary commas, and where would the line be drawn?

Reply:
I agree with deleting these secondaries. I have no idea why they were there. I note that some secondaries are listed in the SMuFL documentation, but not for ~|( . The line would be drawn at some SoPF>3 value.

Reply:
That all seems reasonable enough. I took a note to include such secondary commas in the next pass.

Reply:
OK.